web/tests/Web.Tests/Infrastructure/GivenAnInsightApi.cs

```diff
@@ -22,17 +22,6 @@ public async Task GetSchoolFinancesShouldCallCorrectUrl()
         VerifyCall(HttpMethod.Get, "api/school/123213");
     }
 
-    [Fact]
```

Comment:
Are `BalanceApi` tests going to be included at a later date?

Reply:
Yep - going to do a sweep on tests over the coming weeks,